indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -49,6 +50,9 @@
       :else
       (errors/internal-error! (str "Unknown spatial representation [" coord-sys "]")))))
 
+(defn- update-values [m f & args]
```

Comment:
Will add doctoring to describe it more. but I think update-values is all I need. because I just want to update all the values in a map.